package-lock.json

```diff
@@ -1,22 +1,22 @@
 {
   "name": "rtlcss",
-  "version": "3.0.0",
+  "version": "3.1.0",
```

Comment:
Better use `npm version` when bumping the version so that everything's taken care of including the git tag

Reply:
Thanks 👍